di-ipv-cimit-stub/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+	id "java"
+	id "jacoco"
+}
+
+repositories {
+	mavenCentral()
+}
+
+sourceCompatibility = 17
```

Comment:
We're sticking with java 11 for the lambdas for now aren't we, also we should specify targetCompatibility = 11 as well